packages/components/src/button/style.scss

```diff
@@ -324,6 +324,11 @@
 	.components-visually-hidden {
 		height: auto;
 	}
+
+	// Override for theming on the block mover handle. `.block-editor-block-mover__drag-handle` was needed for specificity to override declaration order.
+	.components-accessible-toolbar &.block-editor-block-mover__drag-handle:focus::before {
+		box-shadow: inset 0 0 0 var(--wp-admin-border-width-focus) $components-color-accent, inset 0 0 0 4px $white;
+	}
```

Comment:
> I'm thinking we should leave all the non-components-package stuff for later so we can address it in a coordinated way.

Do you mean all overrides, including the ones related to the base styles? Or just the overrides needed for other packages too (i.e block editor) ?

Reply:
In this comment I was referring to the latter (other packages like block-editor). But yeah, I think base-styles is better addressed separately as well.

The basic principle I'm trying to stick with is that `Theme` is a wp-components concept, and wp-components is the upstream dependency, so if downstream packages like block-editor want to participate they need to use whatever API that wp-components has provided. What that API should look like is to be decided, but we'll probably need to export `$components-color-accent`, `COLORS.ui.theme`, etc from wp-components.

Reply:
> In this comment I was referring to the latter (other packages like block-editor). But yeah, I think base-styles is better addressed separately as well.
>
> The basic principle I'm trying to stick with is that Theme is a wp-components concept, and wp-components is the upstream dependency, so if downstream packages like block-editor want to participate they need to use whatever API that wp-components has provided.

Agreed on both points.

> What that API should look like is to be decided, but we'll probably need to export `$components-color-accent`, `COLORS.ui.theme`, etc from wp-components.

We'll have to be careful in doing this, since we won't be able to undo these API decisions (because of WordPress backwards compat rules).

I also wonder what is the better format to expose those theme variables — as Sass + Emotion vars, as CSS vars.